packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -460,6 +492,15 @@ export class LitDevTutorial extends LitElement {
     return isNaN(idx) ? undefined : idx;
   }
 
+  /**
+   * Turns an index into a 2-length stringified number for the slug.
+   * @param idx index to slugify
+   * @returns A 2-length stringified number
+   */
+  private _idxToSlug(idx: number): string {
+    return `${idx}`.length == 1 ? `0${idx}` : `${idx}`;
```

Comment:
I think this is the same as:

```js
String(idx).padStart(2, "0");
```

Reply:
darn i merged. Will spin up another PR right now to address